app/jobs/user_session_jobs/send_question_sms_job.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class UserSessionJobs::SendQuestionSmsJob < ApplicationJob
+  queue_as :question_sms
+
+  def perform(user_id, question_id, user_session_id, is_alert = false)
+    user = User.find(user_id)
+
+    return if !is_alert && !user&.sms_notification
```

Comment:
What about predefined participants?
I know it's unlikely for someone to create a predefined participant and disable their ability to send SMS messages. However, there might be cases where a participant requests this, and the researcher wants to achieve this by disabling the SMS option at the predefined participant level.
<img width="625" alt="Screenshot 2024-05-14 at 22 17 49" src="https://github.com/Michigan-State-University/cias-api/assets/61981027/f6fe33f7-e837-4bea-a125-656183884236">